docs/topics/what-is-the-toolkit.adoc

```diff
@@ -11,14 +13,23 @@
 [discrete]
 == What is the {ProductName}?
 
-The {ProductName} ({ProductShortName}) is an extensible and customizable rule-based tool that simplifies the migration of Java applications.
+The {ProductName} ({ProductShortName}) is an extensible and customizable rule-based tool that simplifies the migration and modernization of Java applications.
 
-The {ProductShortName} examines application artifacts, including project source directories and application archives, then produces an HTML report that highlights areas needing changes. {ProductShortName} can migrate Java applications from earlier versions of Red Hat JBoss Enterprise Application Platform or from other application servers, such as Oracle WebLogic Server or IBM WebSphere Application Server.
+{ProductShortName} examines application artifacts, including project source directories and application archives, and then produces an HTML report highlighting areas needing changes. {ProductShortName} supports many migration paths including the following examples:
+
+* Migrating from enterprise application servers to Red Hat JBoss Enterprise Application Platform
+* Containerizing applications and making them cloud-ready
+* Migrating from Spring Boot to Quarkus
+* Updating OpenJDK version
+
+For more information, see link:https://developers.redhat.com/products/mta/use-cases[Migration Toolkit for Applications use cases and migration paths].

```

Comment:
Small change. The fact that this link is not formatted in any way looks odd when you subsequently italicize the "Intro to MTA" guide. I can understand why you are doing it but it still looks weird.

So I think you will need to change the wording to "see the [link] web page" or site or whatever IBM says to use.